WcaOnRails/app/controllers/regional_organizations_controller.rb

```diff
@@ -31,6 +31,25 @@ def update
     end
   end
 
+  def destroy
+    @regional_organization = regional_organization_from_params
+    if current_user.can_manage_regional_organizations?
+      if @regional_organization.start_date.nil?
```

Comment:
Add a separate `is_pending?` method to the RO entity which abstracts away this change. Can also be reused in the HTML view.